lua/nvim-highlight-colors/init.lua

```diff
@@ -22,6 +22,7 @@ local options = {
 	enable_named_colors = true,
 	enable_short_hex = true,
 	enable_tailwind = false,
+	enable_ansi = true,
```

Comment:
Make it false by default, each regex takes quite a bit of performance away, so only standard colours are enabled by default